core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -121,7 +121,7 @@ protected[controller] object RestApiCommons {
  */
 protected[controller] trait RespondWithHeaders extends Directives {
     val allowOrigin = `Access-Control-Allow-Origin`(AllOrigins)
-    val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type")
+    val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type", "User-Agent")
```

Comment:
if the controller log the user agent header, log queries can see which clients and which client versions are accessing the controller.